docs/root/development/performance/battery_impact.rst

```diff
@@ -0,0 +1,63 @@
+.. _dev_performance_battery:
+
+Analysis of battery impact
+==========================
+
+In order to identify how Envoy impacts an application, we have created a control application with modifications to our
+current hello world example applications. To see the actual applications we used, you can go to `iOS Envoy example <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/swift/hello_world>`_,
+`Android control app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/control>`_, `Android Envoy example <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/hello_world>`_.
+
+The current configurations we have are that we make a request every 200ms and disabled caching.
+
+Experimentation method
+~~~~~~~~~~~~~~~~~~~~~~
+
+iOS
+---
+
+* TODO
+
+Android
+-------
+
+Getting the build:
+1. Build the library using ``bazel build android_dist --config=android``
+2. Control: ``bazel mobile-install //examples/kotlin/control:hello_control_kt``
+3. Envoy: ``bazel mobile-install //examples/kotlin/hello_world:hello_envoy_kt --fat_apk_cpu=armeabi-v7a``
+
+Experiment steps:
+1. Set a phone's display to sleep after 30minutes of inactivity
+2. Unplug the phone from any power source
+3. Open up an application
+4. Wait for the phone to sleep
+5. Look at the battery drain from an application like `AccuBattery <https://play.google.com/store/apps/details?id=com.digibites.accubattery&hl=en_US>`_
```

Comment:
Are there any gotchas for using this tool?

Reply:
Not done writing this up but I can continue writing more after we merge this unless you want to wait and/or use this PR for your findiings

Reply:
Sounds good

Reply:
Maybe make a note of the pieces still in progress in these docs though